cluster/gce/gci/configure-helper.sh

```diff
@@ -2286,6 +2286,23 @@ EOF
   fi
 }
 
+# A helper function to set up a custom yaml for a k8s addon.
```

Comment:
A couple real examples would help

Reply:
It is invoked below in line 2456 & 2457.